data/remnant/remnant 2 cognizance.txt

```diff
@@ -867,6 +867,27 @@ mission "Remnant: Cognizance 21"
 			`	The <ship> settles into the current landing pattern for the main spaceport on <destination> as you flip through the exterior camera feeds. In the distance, you notice that one of the research hangars has some activity going on around it, but otherwise, the spaceport seems to be humming along as usual. Dusk stops by the cockpit as you settle the <ship> into its berth. "Thank you for helping us, <first>. It will take us some time to analyze this data, but we will likely be mounting another expedition to Ssil Vida in the near future." He pauses, then adds, "I do not think we ever mentioned it, but, as is appropriate for your assistance, a logistics adjustment of <payment> has been applied to your account to cover your costs and provide resources to prepare for future endeavors." Dusk gives you a polite nod before heading off to help the other researchers unload the equipment.`
 
 
+mission "Remnant: Cognizance Exploration Patch"
+	landing
+	invisible
+	repeat
+	to offer
+		has explorationinterest
+	on offer
+		"exploration interest" += explorationinterest
+		clear explorationinterest
+		fail
+
+mission "Remnant: Cognizance Science Patch"
+	landing
+	invisible
+	repeat
+	to offer
+		has scienceinterest
+	on offer
+		"science interest" += scienceinterest
+		clear scienceinterest
+		fail
```

Comment:
If it's repeating, it doesn't need to be two missions.
Also, there should be three empty lines above and below.
```suggestion

mission "Remnant: Cognizance Interest Condition Patch"
	landing
	invisible
	repeat
	to offer
		or
			has "explorationinterest"
			has "scienceinterest"
	on offer
		"exploration interest" += explorationinterest
		clear "explorationinterest"
		"science interest" += scienceinterest
		clear "scienceinterest"
		fail


```